---
layout: archive
title: Zulip Chat Archive
permalink: /stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf.html
---

<h2>Stream: <a href="https://leanprover-community.github.io/archive/stream/274007-lean-gptf/index.html">lean-gptf</a></h2>
<h3>Topic: <a href="https://leanprover-community.github.io/archive/stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf.html">âœ” unable to use gptf</a></h3>

<hr>

<base href="https://leanprover.zulipchat.com/">

<head><link href="/style.css" rel="stylesheet"></head>

{% raw %}

<a name="252132045"></a>
<h4><a href="https://leanprover.zulipchat.com/#narrow/stream/274007-lean-gptf/topic/%E2%9C%94%20unable%20to%20use%20gptf/near/252132045" class="zl"><img src="https://leanprover-community.github.io/archive/assets/img/zulip2.png" alt="view this post on Zulip"></a> Arthur Adjedj <a href="https://leanprover-community.github.io/archive/stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf.html#252132045">(Sep 06 2021 at 06:26)</a>:</h4>
<p>Turns out I simply don't know how to read, and applied to the openai beta instead of the gptf one. Sorry for bothering you. Have a great day.</p>



<a name="252132051"></a>
<h4><a href="https://leanprover.zulipchat.com/#narrow/stream/274007-lean-gptf/topic/%E2%9C%94%20unable%20to%20use%20gptf/near/252132051" class="zl"><img src="https://leanprover-community.github.io/archive/assets/img/zulip2.png" alt="view this post on Zulip"></a> Notification Bot <a href="https://leanprover-community.github.io/archive/stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf.html#252132051">(Sep 06 2021 at 06:26)</a>:</h4>
<p><span class="user-mention silent" data-user-id="432410">Arthur Adjedj</span> has marked this topic as resolved.</p>



<a name="264178120"></a>
<h4><a href="https://leanprover.zulipchat.com/#narrow/stream/274007-lean-gptf/topic/%E2%9C%94%20unable%20to%20use%20gptf/near/264178120" class="zl"><img src="https://leanprover-community.github.io/archive/assets/img/zulip2.png" alt="view this post on Zulip"></a> Sina <a href="https://leanprover-community.github.io/archive/stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf.html#264178120">(Dec 08 2021 at 16:10)</a>:</h4>
<p><span class="user-mention silent" data-user-id="432410">Arthur Adjedj</span> <a href="#narrow/stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf/near/251935916">said</a>:</p>
<blockquote>
<p>Hi, I tried setting up everything in order to use lean-gptf, but when trying to use the tactic, I end up with the following error message : </p>
<div class="codehilite" data-code-language="Lean"><pre><span></span><code><span class="o">[</span><span class="n">gptf_proof_search_step</span><span class="o">]</span> <span class="n">run_best_beam_candidate</span> <span class="n">UNEXPECTED</span> <span class="n">MESSAGE</span><span class="o">:</span>
<span class="c1">---</span>
<span class="o">[</span><span class="s2">"ERROR { </span><span class="se">\"</span><span class="s2">error</span><span class="se">\"</span><span class="s2">:{ </span><span class="se">\"</span><span class="s2">code</span><span class="se">\"</span><span class="s2">:null, </span><span class="se">\n</span><span class="s2">    </span><span class="se">\"</span><span class="s2">message</span><span class="se">\"</span><span class="s2">:</span><span class="se">\"</span><span class="s2">No such organization: org-kuQ09yewcuHU5GN5YYEUp2hh.</span><span class="se">\"</span><span class="s2">, </span><span class="se">\n</span><span class="s2">    </span><span class="se">\"</span><span class="s2">param</span><span class="se">\"</span><span class="s2">:null, </span><span class="se">\n</span><span class="s2">    </span><span class="se">\"</span><span class="s2">type</span><span class="se">\"</span><span class="s2">:</span><span class="se">\"</span><span class="s2">invalid_request_error</span><span class="se">\"</span><span class="s2">}}"</span><span class="o">]</span>
<span class="c1">---</span>
</code></pre></div>
<p>I tried changing the org name to my personal one, but it lead to a new error saying the engine doesn't exist since it should be tied to the other, first org. What can I do to correct this ? Thanks in advance for the help</p>
</blockquote>
<p>I'm getting the exact same error with the same org id. How did you manage to resolve it? I also followed the same steps of ttps://github.com/jesse-michael-han/lean-gptf.<br>
(By the way, i did not get an email with instruction as advertised on the google form ( <a href="https://bit.ly/3nNWMyB">https://bit.ly/3nNWMyB</a>), but a direct email from Open AI. I have my API_KEY from <a href="https://beta.openai.com/account/api-keys">https://beta.openai.com/account/api-keys</a>.</p>



<a name="265124657"></a>
<h4><a href="https://leanprover.zulipchat.com/#narrow/stream/274007-lean-gptf/topic/%E2%9C%94%20unable%20to%20use%20gptf/near/265124657" class="zl"><img src="https://leanprover-community.github.io/archive/assets/img/zulip2.png" alt="view this post on Zulip"></a> Kunhao Zheng <a href="https://leanprover-community.github.io/archive/stream/274007-lean-gptf/topic/.E2.9C.94.20unable.20to.20use.20gptf.html#265124657">(Dec 16 2021 at 08:32)</a>:</h4>
<p>I've experienced the <code>Engine not found</code> error and explicitly providing the key in the lean file solves that.</p>



{% endraw %}

<hr><p>Last updated: Jan 25 2023 at 00:06 UTC</p>